### PR TITLE
For issue Fenix#10231 update proton icons third batch

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/ui/icons/src/main/res/drawable/mozac_ic_password_reveal.xml [hunk 1]
```diff
@@ -1,18 +1,17 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (gabrielluong) [inline components/ui/icons/src/main/res/drawable/mozac_ic_password_reveal.xml:1]: Undoo this

**Review by gabrielluong — APPROVED** [review]

File: components/ui/icons/src/main/res/drawable/mozac_ic_password_reveal.xml (hunk 1 above)
Comment (Amejia481) [reply components/ui/icons/src/main/res/drawable/mozac_ic_password_reveal.xml:1]: 👍🏽

**Comment by Amejia481** [comment]
@Mergifyio backport releases/75.0

**Comment by Amejia481** [comment]
@Mergifyio backport releases/75.0

**Comment by Amejia481** [comment]
@Mergifyio backport releases/75.0

**Comment by Amejia481** [comment]
@Mergifyio backport releases/75.0

**Pull request merged**